sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -367,6 +409,8 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [cognitive_resource_portal]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account
 [cognitive_resource_cli]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account-cli
 
+[recognize_healthcare_sample]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md
```

Comment:
Our CI recently added a check to validate links. Because this is still not in master, our CI might fail because it can't find the link. What you can do is update your link to the commit, or comment this line and in a next PR uncomment